src/hatch_rest_api/util_bootstrap.py

```diff
@@ -60,15 +64,22 @@ async def get_rest_devices(
     def create_rest_devices(iot_device):
         if iot_device["product"] == "restPlus":
             return RestPlus(
+                device_name=iot_device["name"],
+                thing_name=iot_device["thingName"],
+                shadow_client=shadow_client
+            )
+        elif iot_device["product"] == "riot":
+            return RestIot(
                 device_name=iot_device["name"],
                 thing_name=iot_device["thingName"],
                 shadow_client=shadow_client,
+                favorites=favorites_map[iot_device["macAddress"]]
             )
         else:
             return RestMini(
                 device_name=iot_device["name"],
                 thing_name=iot_device["thingName"],
-                shadow_client=shadow_client,
+                shadow_client=shadow_client
```

Comment:
black styling should add this comma back in, allows additions to a list without conflicts across pr